webapp/src/Controller/PublicController.php

```diff
@@ -100,6 +100,14 @@ public function scoreboardDataZipAction(
         return $this->dj->getScoreboardZip($request, $requestStack, $contest, $this->scoreboardService);
     }
 
+    #[Route(path: '/dynamic-css', name: 'get_dynamic_css')]
```

Comment:
The name is too broad. It should be something like `scoreboardCategoryColorCss` or something.

Reply:
And rename if there are more?

Reply:
More dynamic CSS? I would put them in separate actions, since otherwise you get one big if here, which doesn't make sense IMHO.

Reply:
I'll change it for now, not sure if we have more pages like this so if the discussion will actually ever be non-theoretical.